src/components/common/SpendingLimitLabel/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { type ReactElement } from 'react'
+import { Box, Typography } from '@mui/material'
+import SpeedIcon from '@mui/icons-material/Speed'
+import type { BoxProps } from '@mui/system'
+
+const SpendingLimitLabel = ({
+  label,
+  isOneTime = false,
+  ...rest
+}: { label: string | ReactElement; isOneTime?: boolean } & Omit<BoxProps, 'display' | 'alignItems' | 'gap'>) => {
```

Comment:
Why omit some of the Box props? Seems more complicated than it should be. Let them override them if they want to.